src/engine/strat_engine/backstore/metadata.rs

```diff
@@ -1085,4 +1126,81 @@ mod tests {
             .tests(30)
             .quickcheck(static_header as fn(u64, u32) -> TestResult);
     }
+
+    #[test]
```

Comment:
In final copy, test should have header comment explaining the property it verifies.

Reply:
Done, thanks!